public/index.html

```diff
@@ -9,7 +9,8 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="https://honesty.store/favicon.ico">
+	<link rel="shortcut icon" href="https://honesty.store/favicon.ico">
```

Comment:
why are these tabbed one out?

Reply:
not sure where this came from as those are still only tabbed once but with width 8 - on my local branch they're indented correctly and i've checked the prettier config which sets tabs them width 4.